docs/protocol/Security.md

```diff
@@ -0,0 +1,22 @@
+# Protocol Security
```

Comment:
Should add a section here that talks about roles within the system and how we assume that the governor is not acting maliciously, similar to yearn's section on this role https://docs.yearn.finance/security/index#security-assumption